packages/jellyfish-transaction/src/script/control.ts

```diff
@@ -8,3 +8,57 @@ export class OP_RETURN extends StaticCode {
     super(0x6a, 'OP_RETURN')
   }
 }
+
+export class OP_NOP extends StaticCode {
```

Comment:
You should move ALL your code to the line before OP_RETURN.
Should be in this sequence : 61, 62, 63 .... 69, 6a